src/view/panel/CoordinateMousePositionPanel.js

```diff
@@ -339,11 +339,11 @@ Ext.define('BasiGX.view.panel.CoordinateMousePositionPanel', {
                 var mapCode = me.olMap.getView().getProjection().getCode()
                     .split(':')[1];
                 var filtered = Ext.Array.filter(proj4jObjects, function(obj) {
-                    // Loose equality check since code could be a string or an int
-                    return obj.code === mapCode;
+                    // convert values to strings as they can be either a string or an int
+                    return obj.code.toString() == mapCode.toString();
```

Comment:
If both values are being converted to a string, I would prefer a strict equality check:

```suggestion
                    return obj.code.toString() === mapCode.toString();
```

And just wondering if `obj.code` is defined always? Otherwise `toString()` might fail.

Reply:
Yes, good idea! To also match `undefined` we might want to use `Ext.isDefined` (or whatever is appropriate), something like:

```js
Ext.isDefined(obj.code) && (obj.code.toString() === mapCode.toString())
```